docs/data/charts/styling/styling.md

```diff
@@ -170,6 +170,15 @@ From here, you can target any subcomponents with its class name.
 
 {{"demo": "SxStyling.js"}}
 
+### Drawing area background
+
+To set a background color in the drawing area, you should create a dedicated `<rect />`.
+This is only doable with [composition](/x/react-charts/composition/) because you have to place this new component before all plot component.
```

Comment:
```suggestion
This is only doable with [composition](/x/react-charts/composition/) because you have to place this new component before all plot components.
```